locales/it/index.yml

```diff
@@ -1715,10 +1714,6 @@ wallet:
       SUCCESS:
         title: Hai pagato {{amount}}
         button: Ok, chiudi
-        banner:
-          title: Puoi dirci com’è andata?
-          content: Raccontaci la tua esperienza con il pagamento e aiutaci a migliorare.
-          action: Vai al sondaggio
```

Comment:
This must not be removed. This should be part of `WalletPaymentFeebackBanner`. Could you update it with proper keys?